src/schedule/ll_schedule.c

```diff
@@ -160,28 +153,27 @@ static void schedule_ll_task_done(struct ll_schedule_data *sch,
 /* perf measurement windows size 2^x */
 #define CHECKS_WINDOW_SIZE	10
 
-static inline void dsp_load_check(struct ll_dsp_load *load, uint32_t cycles0, uint32_t cycles1)
+static inline void dsp_load_check(struct task *task, uint32_t cycles0, uint32_t cycles1)
 {
-	uint32_t diff, max;
+	uint32_t diff;
 
 	if (cycles1 > cycles0)
 		diff = cycles1 - cycles0;
 	else
 		diff = UINT32_MAX - cycles0 + cycles1;
 
-	load->cycles_sum += diff;
+	task->cycles_sum += diff;
 
-	if (load->cycles_peak < diff)
-		load->cycles_peak = diff;
+	if (task->cycles_max < diff)
+		task->cycles_max = diff;
```

Comment:
Why max instead of peak? Max suggests range and here we clearly want to track 'peak' value in given timespan

Reply:
*_eak was my name for variable before the per task implementation. With the per task implementation I decided to reuse the same struct task members as Zephyr side implementation does. The struct task members have different names.